test/utils/update/fetch.beneficial.owners.private.data.spec.ts

```diff
@@ -0,0 +1,246 @@
+import {
+  fetchBeneficialOwnersPrivateData
+} from "../../../src/utils/update/fetch.beneficial.owners.private.data";
+
+jest.mock('../../../src/service/private.overseas.entity.details');
+jest.mock('../../../src/utils/logger');
+jest.mock("../../../src/utils/feature.flag" );
+
+import { describe, expect, jest, test } from '@jest/globals';
+import { Request } from "express";
+import { ApplicationData } from '../../../src/model';
+import { getBeneficialOwnersPrivateData } from '../../../src/service/private.overseas.entity.details';
+import { logger } from '../../../src/utils/logger';
+import {
+  PRIVATE_BO_DATA_MOCK,
+  PRIVATE_BO_ADDRESS
+} from "../../__mocks__/get.beneficial.owner.private.data.mock";
+import { isActiveFeature } from "../../../src/utils/feature.flag";
+
+const mockGetBeneficialOwnersPrivateData = getBeneficialOwnersPrivateData as jest.Mock;
+const mockLoggerInfo = logger.info as jest.Mock;
+const mockLoggerError = logger.errorRequest as jest.Mock;
+const mockIsActiveFeature = isActiveFeature as jest.Mock;
+
+describe("Test fetching and mapping BO private data", () => {
+  let appData: ApplicationData, req: Request;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("should fetch and map BO private data", async () => {
+    appData = {
```

Comment:
take out as mock object

Reply:
a session mock or move it up to the top of the page?

Reply:
either would help